apps/src/music/blockly/toolbox/toolboxBlocks.ts

```diff
@@ -250,6 +283,7 @@ const toolboxBlocks: {[blockType in BlockTypes | string]: BlockInfo} = {
     },
   },
   ['math_arithmetic']: {
+    text: 'Arithmetic',
```

Comment:
There are a couple blocks like this that are hard to figure out what text would be most helpful to curriculum writers. This block just has two number inputs separated by a dropdown field (with options + - × ÷ ^). Would it be helpful to show those options on the label so it’s clearer which block this is?  Similar question for logic_compare (which allows for a variety of equality and comparison operators) and math_number_property (with options like even, odd, prime, whole, positive, negative, divisible by).


This isn’t a blocker however and could be something we ask curriculum for help with once this available on levelbuilder.

Reply:
Good question. I think yeah it might be good to get feedback from levelbuilders once they start using the tool and we can update accordingly.